packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -115,11 +115,40 @@ export async function generateImageData({
       : DEFAULT_BREAKPOINTS
   }
 
-  const {
-    fit = `cover`,
-    cropFocus = sharp.strategy.attention,
-    duotone,
-  } = transformOptions
+  const { fit = `cover`, duotone } = transformOptions
+
+  let cropFocus = transformOptions.cropFocus || `attention`
+
+  const cropFocusOptions = [
+    `top`,
+    `right top`,
+    `right`,
+    `right bottom`,
+    `bottom`,
+    `left bottom`,
+    `left`,
+    `left top`,
+    `north`,
+    `northeast`,
+    `east`,
+    `southeast`,
+    `south`,
+    `southwest`,
+    `west`,
+    `northwest`,
+    `center`,
+    `centre`,
+    `entropy`,
+    `attention`,
+  ]
```

Comment:
Can we change it into the following? This makes the link more dynamic as it gets it values from sharp so we don't have to maintain it. Could you also add a unit test to test this behavior?
 
```suggestion
  const cropFocusOptions = Object.keys(sharp.position).concat(Object.keys(sharp.gravity)).concat(Object.keys(sharp.strategy))
```

Reply:
Thanks. `sharp.position` doesn't actually exist, but `sharp.gravity` and `sharp.strategy` do, so I had to keep the hardcoded values for position:
```ts
const cropFocusOptions = [
  `top`,
  `right top`,
  `right`,
  `right bottom`,
  `bottom`,
  `left bottom`,
  `left`,
  `left top`,
]
  .concat(Object.keys(sharp.gravity))
  .concat(Object.keys(sharp.strategy))
```

Reply:
I can also make a unit test.

Reply:
Interesting cause for me this works:
```
$ node -e "console.log(Object.keys(require('sharp').position))"
[
  'top',
  'right',
  'bottom',
  'left',
  'right top',
  'right bottom',
  'left bottom',
  'left top'
]
```

Unit tests are always great 😍 But feel free to do it in a follow up

Reply:
<img width="885" alt="Screen Shot 2021-09-13 at 4 42 15 PM" src="https://user-images.githubusercontent.com/50968964/133153514-95ce1eaf-d7ed-46f1-8036-0f6e1be0b08c.png">

I think it's just a typescript problem that can just be ignored. I'll change it back to `Object.keys(sharp.position)`. Also I'll try to add the unit test in another pull request sometime soon.